site/assets/scss/_content.scss

```diff
@@ -41,6 +41,10 @@
       overflow-x: auto;
     }
 
+    thead {
+      border-bottom: 2px solid currentColor;
```

Comment:
You can propose a modification in Bootstrap. In Boosted I'll let it as it is not because I disagree but because it is easier to update when new commits come back from Bootstrap. I tend to report their mistakes as well to have the closest code between Bootstrap and Boosted.